src/utils/checkConnectivityInterval.ts

```diff
@@ -1,6 +1,5 @@
-// @flow
+let interval: number | null = null;
```

Comment:
why do we need `number | null`?

`let interval: number = null` this has to be enough.

Reply:
This doesn't let me assign null as null is not of type number.

Reply:
strictNullChecks=true. Ok